recipes/flatbush/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+CMAKE_MINIMUM_REQUIRED(VERSION 3.12)
+
+PROJECT(test_package VERSION 1.0.0 LANGUAGES CXX)
+INCLUDE(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+SET(CMAKE_CXX_STANDARD 11)
+ADD_DEFINITIONS("-DMINIMAL_SPAN")
```

Comment:
I wasn't sure if this was possible at all before creating my PR but the thought of leaving the burden to the user was bothering me. I did have to put in some guards (i.e. force cppstd=11 in a few cases) for the test_package to pass CI:

- Although GCC 5 has support for C++11, it doesn't default use it by default.
- Apple's Clang appears to default to C++98 and cppstd is not set at all.

I believe this shouldn't interfere with the user's setup since CXX version from CMakelists would take precedence over the profile settings.

Reply:
It's a bit complicated, but still possible. Let's leave it as it is.